pkg/virtctl/imageupload/imageupload.go

```diff
@@ -141,16 +143,19 @@ func NewImageUploadCommand(clientConfig clientcmd.ClientConfig) *cobra.Command {
 
 func usage() string {
 	usage := `  # Upload a local disk image to a newly created DataVolume:
-  {{ProgramName}} image-upload dv dv-name --size=10Gi --image-path=/images/fedora30.qcow2
+  {{ProgramName}} image-upload dv fedora-dv --size=10Gi --image-path=/images/fedora30.qcow2
 
   # Upload a local disk image to an existing DataVolume
-  {{ProgramName}} image-upload dv dv-name --no-create --image-path=/images/fedora30.qcow2
+  {{ProgramName}} image-upload dv fedora-dv --no-create --image-path=/images/fedora30.qcow2
+
+  # Upload a local disk image to a newly created PersistentVolumeClaim
+  {{ProgramName}} image-upload pvc fedora-pvc --size=10Gi --image-path=/images/fedora30.qcow2
 
   # Upload a local disk image to an existing PersistentVolumeClaim
-  {{ProgramName}} image-upload pvc pvc-name --image-path=/images/fedora30.qcow2
+  {{ProgramName}} image-upload pvc fedora-pvc --no-create --image-path=/images/fedora30.qcow2
```

Comment:
I want to make sure we aren't breaking someone's previous usage of this command. What's the expectation if this command was executed without `--no-create` yet the pvc already existed?

Reply:
It was always and still is the case that `--no-create` will use an existing PVC.

Reply:
but what about when the `--no-create` command arg isn't used. is the default behavior changing with this pr?

Reply:
Yes but that was the whole idea.  `image-upload pvc foo ...` currently implies --no-create and we do not want that.  See bz.

`--no-create` semantics are still the same as ever for legacy invocation `image-upload pvc-name=foo ...`. and `image-upload dv foo`

Reply:
ah, got it